code/development-env/README.md

```diff
@@ -164,13 +134,13 @@ Therefore, the URL of the auth service is different for the infrastructure servi
 The IP address that is used in the ingress rule can be configured using the following environment variable
 
 ```bash
-AUTHORISATION_SERVICE_FOR_INGRESS=<ip to access auth service>
+AUTHORISATION_SERVICE_FOR_INGRESS=10.0.2.2:9000
 ```
 
 where `<ip-to-access-auth-service>` needs to be configured to be the IP address and port through which a service in the cluster can access the authorisation service running on `localhost`.
 From VirtualBox, this is expected to be `http://10.0.2.2:9000` as `10.0.2.2` is the IP through which items running in VirtualBox can access the host machine, and the auth service is configured to run on port `9000` by default.
```

Comment:
The text in the following paragraph needs to be updated if you think we should have the IP address given in the environment variable. When I wrote this I was trying to keep it generic so that someone who didn't end up with a VirtualBox gateway back to the host at `10.0.2.2` might be able to understand the problem and find the solution.

Reply:
Actually, I think because AUTHORISATION_SERVICE_FOR_INGRESS is defined here:
https://github.com/NERC-CEH/datalab/blob/master/code/development-env/docker/docker-compose-app.yml#L6
without a possible override (c.f. LOG_LEVEL) then I don't think what's set in the environment variable makes any difference!

Reply:
I'll make an update :-)